### PR TITLE
Added details to the README.md

### PR DESCRIPTION
**Pull request by omarspx**
Added details on how to access the created wallet when using "./create_wallet.sh" or "./docker_run_and_create_wallet.sh".
Added details on how to get your docker-machine IP (It doesn't work with localhost or if you use the docker container IP).
Added a note about copying the wallets from docker image to gui (It's not necessary if you use the commands above).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,6 +36,11 @@ You can now claim the initial NEO and GAS:
 
 `./create_wallet` will display several internal error messages, which is expected as long as at the end you still get a success message.
 
+If you call ./create_wallet.sh or ./docker_run_and_create_wallet.sh, it will create 2 files in your current directory :
+neo-privnet.wallet : a wallet you can use with neo-python
+neo-privnet.wif : a wif private key you can import into other clients (neo-gui for exemple).
+
+Those files will get you access to the wallet containing all the NEOs of your private network.
```
Comment (AshRolls) [inline README.md]: markdown formatting is messed up here, it's treating this line as a sub-heading

**Comment by metachris** [comment]
Here is a website where you can paste markdown and see the resulting HTML: https://jbt.github.io/markdown-editor

There is errors in this PR, the subheader like @AshRolls mentioned, and the points above should perhaps be bullet points.

**Review by metachris — CHANGES_REQUESTED** [review]
markdown is not alright

File: README.md [hunk 2]
```diff
@@ -62,7 +67,15 @@ If you copy the protocol.json file from the configs directory of this repo and r
...
 Change each occurrence of 127.0.0.1 to the IP of the system or vm running your docker image.
+If you use docker, to find the name of your machine, type :
+
+    docker-machine ls
+
+And get the ip with :
 
+    docker-machine ip "Nameofyourmachine"
+
+(By default, the machine name is "default").
```
Comment (AshRolls) [inline README.md]: These steps are only useful in windows for Windows Home users who don't have hyper-v (so use docker-machine).  It should be put into a sub-section to make that distinction clear

File: README.md [hunk 3]
```diff
@@ -36,6 +38,15 @@ You can now claim the initial NEO and GAS:
 
 `./create_wallet` will display several internal error messages, which is expected as long as at the end you still get a success message.
 
+If you call ./create_wallet.sh or ./docker_run_and_create_wallet.sh, it will create 2 files in your current directory :
```
Comment (AshRolls) [inline README.md]: ./create_wallet.sh and ./docker_run_and_create_wallet.sh should both be shown with code markup ``

File: README.md [hunk 4]
```diff
@@ -10,6 +10,8 @@ See the section below on extracting Neo and Gas as the private chain in this doc
 You will also need to install and configure the neo-gui pc client on your favorite distro. This involves editing the protocol.json file to point the seeds at your docker IP addresses.
 
 
+
```
Comment (metachris) [inline README.md]: don't add so many empty lines

File: README.md [hunk 5]
```diff
@@ -36,6 +38,15 @@ You can now claim the initial NEO and GAS:
 
 `./create_wallet` will display several internal error messages, which is expected as long as at the end you still get a success message.
 
+If you call ./create_wallet.sh or ./docker_run_and_create_wallet.sh, it will create 2 files in your current directory :
+
+- neo-privnet.wallet : a wallet you can use with neo-python
```
Comment (metachris) [inline README.md]: wrap filenames in code markup ``

File: README.md [hunk 6]
```diff
@@ -36,6 +38,15 @@ You can now claim the initial NEO and GAS:
...
 `./create_wallet` will display several internal error messages, which is expected as long as at the end you still get a success message.
 
+If you call ./create_wallet.sh or ./docker_run_and_create_wallet.sh, it will create 2 files in your current directory :
+
+- neo-privnet.wallet : a wallet you can use with neo-python 
+- neo-privnet.wif : a wif private key you can import into other clients (neo-gui for exemple).
+ 
+Those files will get you access to the wallet containing all the NEOs of your private network.
+
+
```
Comment (metachris) [inline README.md]: only have 1 blank line here

File: README.md [hunk 7]
```diff
@@ -62,15 +73,30 @@ If you copy the protocol.json file from the configs directory of this repo and r
 ],
 
 Change each occurrence of 127.0.0.1 to the IP of the system or vm running your docker image.
-
```
Comment (metachris) [inline README.md]: 1 blank line should be here

File: README.md [hunk 8]
```diff
@@ -62,15 +73,30 @@ If you copy the protocol.json file from the configs directory of this repo and r
...
+And get the ip with :
+
+    docker-machine ip "Nameofyourmachine"
+
+(By default, the machine name is "default"). Use this ip to replace each occurence of 127.0.0.1 in the SeedList array.
+
+
 ## Copy wallets from docker image to neo-gui
 
+Note : You won't need this step if you used ./create_wallet.sh or ./docker_run_and_create_wallet.sh in the previous step (The multiparty signature and neo/gas extraction should already be done).
```
Comment (metachris) [inline README.md]: wrap commands in code markup ``

File: README.md [hunk 9]
```diff
@@ -36,11 +35,17 @@ You can now claim the initial NEO and GAS:
 
 `./create_wallet` will display several internal error messages, which is expected as long as at the end you still get a success message.
 
+If you call ./create_wallet.sh or ./docker_run_and_create_wallet.sh, it will create 2 files in your current directory :
+
+- `neo-privnet.wallet` : a wallet you can use with neo-python 
+- `neo-privnet.wif` : a wif private key you can import into other clients (neo-gui for exemple).
```
Comment (metachris) [inline README.md]: remove whitespace before colon. also `exemple` should be `example`.

File: README.md (hunk 3 above)
Comment (metachris) [reply README.md]: Still todo

File: README.md (hunk 7 above)
Comment (metachris) [reply README.md]: Still todo

File: README.md [hunk 10]
```diff
@@ -36,11 +35,17 @@ You can now claim the initial NEO and GAS:
 
 `./create_wallet` will display several internal error messages, which is expected as long as at the end you still get a success message.
 
+If you call `./create_wallet.sh` or `./docker_run_and_create_wallet.sh`, it will create 2 files in your current directory :
```
Comment (metachris) [inline README.md]: remove the space before the colon here please.

File: README.md [hunk 11]
```diff
@@ -36,11 +35,17 @@ You can now claim the initial NEO and GAS:
...
+
+- `neo-privnet.wallet`: a wallet you can use with neo-python 
+- `neo-privnet.wif`: a wif private key you can import into other clients (neo-gui for example).
+ 
+Those files will get you access to the wallet containing all the NEOs of your private network.
```
Comment (metachris) [inline README.md]: "all the NEO and GAS for your private network" please

**Comment by metachris** [comment]
made 2 last comments, then we can merge

File: README.md (hunk 10 above)
Comment (omarspx) [reply README.md]: I will, hope you didn't mind the extra work, will do better next time =)

File: README.md (hunk 11 above)
Comment (omarspx) [reply README.md]: Technically, the printed output is :
Finished, Ae9MUYt44Le4ux2Hcd8JEnDpfacN7xQCgD should now own all the NEO on the private network.

Then it claims 48 Gas :
Claiming the GAS...

---------------------------------------------------------------
Will make claim for 48.0 GAS
------------------------------------------------------------------

**Review by metachris — APPROVED** [review]

**Pull request merged**